site/bg.njk

```diff
@@ -0,0 +1,145 @@
+---
+layout: default
+meta_title: кампания за спирането на пандемията COVID-19 
+meta_description: Нашите правителства не могат да се справят без нашата помощ, трябва да предприемем действия сега! Изпълнете своята роля и останете у дома мамка му!
+body_class: home
+title: '#StayTheFuckHome'
+lang: en
+---
+
+<main id="main" class="main inner flex flex-1 flex-col py-10 lg:py-20 focus:outline-none" tabindex="-1">
+
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">Кампания за спирането на пандемията COVID-19</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                Нашите правителства прилагат мерки за контрол на разпространението на
+                <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
```

Comment:
Let's use the BG version of the article -- [Here](https://bg.wikipedia.org/wiki/SARS-CoV-2)